scratch/PCA/extract_data.ipynb

```diff
@@ -18,40 +18,81 @@
     "\"\"\"\n",
     "\n",
     "from __future__ import division\n",
```

Comment:
Yes. I’m in my office.

> On Aug 16, 2016, at 11:01 AM, HGeerlings <notifications@github.com> wrote:
> 
> In scratch/PCA/extract_data.ipynb <https://github.com/csm-adapt/tracr/pull/4#discussion_r74976256>:
> 
> > @@ -18,40 +18,81 @@
> >      "\"\"\"\n",
> >      "\n",
> >      "from __future__ import division\n",
> Sure Branden,
> 
> I can meet at 11:15, would that work for you?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/csm-adapt/tracr/pull/4/files/c34dbfd1457b395e51fb01c9960400ba80f95fbc#r74976256>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AFjEmRkrgnsPPTOpTYPOy43qCZgect5pks5qgez7gaJpZM4Jk3y6>.
>